### DISCUSSION TITLE
CSS Vars Unitless

### DISCUSSION THREAD

**Discussion by MrX8503** [tailwindlabs/tailwindcss#2475] [Ideas]
I want to be able to do this:

```
addBase({
    ':root': {
        '--scale': 4
    }
});
```

But I get this for the output:
```
:root {
    --scale: 4px
}
```

**Comment by adamwathan** [comment]
Wrap the number in quote and you should get the result you want:

```
addBase({
  ':root': {
    '--scale': '4',
  },
})
```

Looks like [postcss-js](https://github.com/postcss/postcss-js) which we use under the hood automatically converts bare numbers to pixel values.
